src/main/java/org/wildfly/security/auth/realm/ldap/LdapSecurityRealm.java

```diff
@@ -130,6 +135,7 @@
         this.credentialLoaders = credentialLoaders;
         this.credentialPersisters = credentialPersisters;
         this.evidenceVerifiers = evidenceVerifiers;
+        this.name = name;
```

Comment:
I am not sure if this is not unsystematic - something like keeping name of realm/keystore/credentialstore etc. would be useful on a lot of other places too - I think we should add it to all of not at all...